.github/workflows/native-handler-dev-stage-publish.yml

```diff
@@ -0,0 +1,66 @@
+name: Publish AsyncAPI native handler library to dev/stage central 
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: Select Environment
+        required: true
+        options:
+          - DEV CENTRAL
+          - STAGE CENTRAL
+
+jobs:
+  publish-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      
+      - name: Set up JDK 21
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: 21.0.3
+
+      - name: Build the Package
+        env:
+          GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+          packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+          packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+        run: |
+          ./gradlew build -x test
+
+      - name: Set Up Ballerina
+        uses: ballerina-platform/setup-ballerina@v1.1.3
+        with:
+          version: 2201.11.0
```

Comment:
Can't we get this lang version from the `gradle.properties` file?

Reply:
This is a temporary fix, will do a proper fix with the async-api tool revamp.